assignments/callbacks.js

```diff
@@ -5,65 +5,78 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 /* 
 
   // GIVEN THIS PROBLEM:
-
   function firstItem(arr, cb) {
     // firstItem passes the first item of the given array to the callback function.
+
   }
 
   // SOLUTION:
-
   function firstItem(arr, cb) {
     return cb(arr[0]);
   }
 
   // NOTES ON THE SOLUTION:
-
   // firstItem is a higher order function.
   // It expects a callback (referred to as `cb`) as its second argument.
   // To test our solution, we can use the given `items` array and a variety of callbacks.
   // Note how callbacks can be declared separately, or inlined.
 
   // TEST 1 (inlined callback):
-
   const test1 = firstItem(items, item => `I love my ${item}!`);
   console.log(test1); // "I love my Pencil!"
 
   // TEST 2 (declaring callback before hand):
-
   function logExorbitantPrice(article) {
     return `this ${article} is worth a million dollars!`;
   };
 
   const test2 = firstItem(items, logExorbitantPrice);
   console.log(test2); // "this Pencil is worth a million dollars!"
+
 */
 
 
+
+
+
 function getLength(arr, cb) {
-  // getLength passes the length of the array into the callback.
+    // getLength passes the length of the array into the callback.
+    return cb(arr.length);
 }
+const output1 = getLength(items, item => `The length is ${item}`);
+console.log(output1);
 
 function last(arr, cb) {
-  // last passes the last item of the array into the callback.
+    // last passes the last item of the array into the callback.
+    return cb(arr.length - 1);
```

Comment:
This is only passing the length of the array minus one. To properly pass the last item of the array, you need to use that equation as the index you're accessing in the array. 

`return cb(arr[arr.length - 1]);`